### PR TITLE
[android] Add sensor clusters screen

### PR DESCRIPTION
**Pull request by Damian-Nordic**
Add a screen for reading and displaying data from remote
sensor devices such as: temperature, relative humidity and
pressure (more can be easily added when needed).

Sample screenshot (with fake data)
image

### CODE REVIEW

File: src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/SensorClientFragment.kt [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+    private val CLUSTERS = mapOf(
+        "Temperature" to mapOf(
+            "read" to { deviceId: Long, endpointId: Int, callback: ReadCallback ->
+              val device = ChipClient.getDeviceController().getDevicePointer(deviceId)
+              val cluster = ChipClusters.TemperatureMeasurementCluster(device, endpointId)
+              cluster.readMeasuredValueAttribute(callback)
+            },
+            "unitValue" to 0.01,
+            "unitSymbol" to "*C"
```
Comment (kkasperczyk-no) [inline src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/SensorClientFragment.kt:182]: I'm just curious, isn't it possible to print degree character ° by something like "\370"?

Comment (Damian-Nordic) [reply src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/SensorClientFragment.kt:182]: I'll try.

**Review by kkasperczyk-no — APPROVED** [review]
Generally it looks pretty good, but I have two remarks to consider. Maybe it would be good idea to display bigger range of data in the X axis, as 15 s is quite short period of time to observe, especially regarding temperature data changes that are pretty slow. Another thing is that this error log may worry someone a little :smiley: If it's not something serious, can we ignore it somehow, or filter serious errors?

**Comment by Damian-Nordic** [comment]
> Generally it looks pretty good, but I have two remarks to consider. Maybe it would be good idea to display bigger range of data in the X axis, as 15 s is quite short period of time to observe, especially regarding temperature data changes that are pretty slow. Another thing is that this error log may worry someone a little smiley If it's not something serious, can we ignore it somehow, or filter serious errors?

The error messages should not normally be displayed. I only captured the sample screenshot using fake data, not being connected to any device and hence the error. Regarding the range, you're right. I can extend it a little. Generally we currently support up to 20 samples collected every 2 seconds. A user must adjust these values. The screenshot only shows the first 15 seconds, but even now the capacity is a bit bigger.

**Review by LuDuda — APPROVED** [review]
This is really good!

**Pull request closed without merging**